### DISCUSSION TITLE
Same Windows Position/Size for Next/Prev Display

### DISCUSSION THREAD

**Discussion by Omanuar** [rxhanson/Rectangle#899] [Ideas]
Next/Prev display shortcut default to center the moving window (v0.58).
I think a must have  applying Same Windows Position/Size( % for different display resolution) for Next/Prev Display BY DEFAULT .
and option for Next/Prev to center center,Next/Prev to bottom center,Next/Prev to top center...

**Comment by finsterwalder** [comment]
I'm totally for moving the window to the same position when switching to another screen and not moving it to the center every time.
An option to move it to center/center would be fine with me. More is superfluous I think.
